examples/knative/go/kn-go-tagging/tagging/tagging.go

```diff
@@ -0,0 +1,139 @@
+package tagging
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/cloudevents/sdk-go/v2/client"
+	"github.com/cloudevents/sdk-go/v2/event"
+	"github.com/cloudevents/sdk-go/v2/protocol"
+	cehttp "github.com/cloudevents/sdk-go/v2/protocol/http"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/vapi/rest"
+	"github.com/vmware/govmomi/vapi/tags"
+	"github.com/vmware/govmomi/vim25/types"
+	"go.uber.org/zap"
+	"knative.dev/pkg/logging"
+)
+
+// Client allows connecting to vSphere APIs and CloudEvent methods.
```

Comment:
nit: code doc should not describe what it does internally but what it exposes, e.g. running a CloudEvents receiver and perform tagging operations.